model/src/user/current_user_guild.rs

```diff
@@ -0,0 +1,74 @@
+use crate::guild::Permissions;
+use crate::id::GuildId;
+use serde::{Deserialize, Serialize};
+
+/// Information about a guild the current user is in.
```

Comment:
Add a note that this is a partial guild used for the "Get Current User Guilds" endpoint

Reply:
Added